pkg/cmd/container/restart.go

```diff
@@ -35,7 +35,7 @@ func Restart(ctx context.Context, client *containerd.Client, containers []string
 			if found.MatchCount > 1 {
 				return fmt.Errorf("multiple IDs found with provided prefix: %s", found.Req)
 			}
-			if err := containerutil.Stop(ctx, found.Container, options.Timeout); err != nil {
+			if err := containerutil.Stop(ctx, found.Container, options.Timeout, "SIGTERM"); err != nil {
```

Comment:
do you plan to add the same flag for `restart`? https://docs.docker.com/reference/cli/docker/container/restart/#signal

(not required, and not necessarily done in the same PR)

Reply:
Yes, that will be a follow up PR